.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "nuget"
+    directory: "/gremlin-dotnet"
+    schedule:
+      interval: "daily"
+    target-branch: "3.5-dev"
+    labels:
+      - ".NET"
```

Comment:
I think that using these two labels can make it easier to filter the PRs, e.g,, by GLV. We'll need to create this label though before merging the PR.